src/streamlink_cli/utils/__init__.py

```diff
@@ -7,10 +8,11 @@
 from .progress import progress
 from .player import find_default_player
 from .stream import stream_to_url
+from .lazy_formatter import *
```

Comment:
Should avoid `*` here too. In this case it will also import `string` - which you probably didn't expect...

Reply:
I did not expect that, no.